### ISSUE TITLE
Personal Finance needs angular and core

### ISSUE THREAD

**Issue by rdlauer** [NativeScript/code-samples#84]
https://play.nativescript.org/?template=play-vue&id=4JhSZz&v=49

**Comment by tylerablake** [comment]
I'm working on adding angular to this sample.

**Comment by tylerablake** [comment]
Here's the angular addition: https://play.nativescript.org/?template=play-ng&id=zYpU71&v=8

Let me know if you see any issues.

The 3rd tab was not working on the sample so I tried to match what I thought the sample was trying to do. I'll open a different issue for that though.

**Comment by rdlauer** [comment]
Looks good! I did see one error show up on Android:

`[Moto G (5) Plus]: Error in reading bitmap - java.io.FileNotFoundException: /data/data/org.nativescript.preview/files/app/images/yellow (No such file or directory)`

Any ideas why the taps are so sluggish to navigate between the views?

**Comment by tylerablake** [comment]
Oh, oops, I left my Android test device at the office so I can fix that Android error on Monday.

Is it only sluggish on Android or do you notice it on iOS as well? If you are noticing it on iOS as well I can look at it tomorrow.

**Comment by rdlauer** [comment]
Yeah it seemed pretty slow on iOS as well - at least taps on the tabs aren't registering quickly? Strange.

**Comment by rdlauer** [comment]
I think that was it, thanks!

**Comment by tylerablake** [comment]
Perfect, I'll look into that Android issue tomorrow and post an updated link for that.
